src/main/resources/dump.sql

```diff
@@ -0,0 +1,15 @@
+create database hw0307
+    with owner maksymskripai;
+
+create table users
```

Comment:
try to use correct sql syntax, for example:
https://www.w3schools.com/sql/default.asp

Reply:
its auto created dump for my db, just show what inside...